src/main/frontend/src/components/Header.module.css

```diff
@@ -29,14 +29,13 @@
 .socialmediafit {
   margin-top: auto;
   margin-bottom: auto;
-  margin-right: 50%;
+  margin-left: 65px;
```

Comment:
Adding margin like this to position container will not work as expected. As it is is now header and footer do not have same margins on left nor right side. 
<img width="1528" alt="Screenshot 2021-11-04 at 09 51 37" src="https://user-images.githubusercontent.com/57901159/140285270-a6475bd8-e77a-4527-be88-2d580797b8ae.png">

I suggest adding separate component called LayoutContainer that will be used in both header footer nav and main content. and will wrap content in those components, and will have width 1440px;, margin 0 auto;